src/com/cloudogu/ces/cesbuildlib/Changelog.groovy

```diff
@@ -0,0 +1,32 @@
+package com.cloudogu.ces.cesbuildlib
```

Comment:
Is this file formatted using tabs? :thinking:
Please use spaces. If you want, add an `.editorconfig`  while you're at it.

Could we also get unit test for this file?